kernels/generate_angle_matrix_2d_x.cl

```diff
@@ -0,0 +1,29 @@
+
+__kernel void generate_angle_matrix(
+    IMAGE_dst_matrix_TYPE dst_matrix,
+    IMAGE_src_point_list1_TYPE src_point_list1,
+    IMAGE_src_point_list2_TYPE src_point_list2
+) {
+  const sampler_t sampler = CLK_NORMALIZED_COORDS_FALSE | CLK_ADDRESS_CLAMP_TO_EDGE | CLK_FILTER_NEAREST;
```

Comment:
In new kernel, sampler_t are defined inside kernel. Instead of outside the kernel in the older one. I'm pretty sure it has no impact, it just trigger my OCD :)